notebooks/Zero_cost_Zero_time_Zero_shot_Financial_Sentiment_Analysis.ipynb

```diff
@@ -0,0 +1,2428 @@
+{
```

Comment:
<p>would recommend versioning the key libraries to avoid issues with breaking changes in the future</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/setfit/pull/497/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>